CIP-0010/CIP-0010.md

```diff
@@ -45,16 +45,15 @@ Creating a registry for `transaction_metadatum_label` values has the following b
 
 ## Specification
 
-These are the registered `transaction_metadatum_label` values
-
-transaction_metadatum_label | description
-----------------------------|-----------------------
-0 - 15                      | reserved*
-1967                        | nut.link metadata oracles registry
```

Comment:
@mmahut please rebase the PR so it can be merged, let's keep changes other PRs added